src/universal_transfer_operator/integrations/fivetran.py

```diff
@@ -139,6 +139,8 @@ def to_dict(self) -> dict:
 class FivetranIntegration(TransferIntegration):
     """Fivetran integration to transfer datasets using Fivetran APIs."""
 
+    OPTIONS_CLASS = FiveTranOptions
```

Comment:
What would happen if there are multiple option classes available?

Reply:
Can point out a use case where this would happen?

Reply:
Why would you have multiple types of `FivetranOptions`??

Reply:
@sunank200 and I synced offline and concluded that we can modify the `src/universal_transfer_operator/integrations/__init__.py method - get_ingestor_option_class` when there is a requirement to change the logic for picking the right class for transfer mode - `third-party`.

Also, we are assuming that there will only be in kind of options class required at a time.